core/src/main/scala/io/finch/internal/currentTime.scala

```diff
@@ -1,12 +1,12 @@
 package io.finch.internal
 
+import java.time.{Instant, ZoneId}
 import java.time.format.DateTimeFormatter
-import java.time.Instant
-import java.time.ZoneOffset
+
 
 object currentTime {
   private[this] val formatter: DateTimeFormatter =
-    DateTimeFormatter.RFC_1123_DATE_TIME.withZone(ZoneOffset.UTC)
+    DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss zzz").withZone(ZoneId.of("GMT"))
```

Comment:
Should we also set a `Locale` here? (Looks like [http4s does it](https://github.com/http4s/blaze/blob/7701fe61d826cf2bc842e34e9f6d17c9694f1b74/http/src/main/scala/org/http4s/blaze/http/util/HeaderTools.scala#L13)).